sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/ScannerForMSBuild.java

```diff
@@ -42,6 +47,9 @@ public class ScannerForMSBuild extends Build<ScannerForMSBuild> {
   static final int DOT_NET_CORE_INTRODUCTION_MAJOR_VERSION = 4;
   static final int DOT_NET_CORE_INTRODUCTION_MINOR_VERSION = 1;
   public static final String DOT_NET_CORE_INTRODUCTION_VERSION = DOT_NET_CORE_INTRODUCTION_MAJOR_VERSION + "." + DOT_NET_CORE_INTRODUCTION_MINOR_VERSION;
+  public static final String LATEST_RELEASE = "LATEST_RELEASE";
+  private static final Logger LOG = LoggerFactory.getLogger(ScannerForMSBuild.class);
+  private final GitHub gitHub;
```

Comment:
Nitpick: I would leave a space between the `static` fields defined above and this field, which is not static, and optionally remove the space between this `final` field and the following instance fields.

That would keep the grouping logic of fields as it was before.